scala-package/mxnet-demo/java-demo/README.md

```diff
@@ -1,38 +1,43 @@
 # MXNet Java Sample Project
 This is an project created to use Maven-published Scala/Java package with two Java examples.
 ## Setup
-Please copy the downloaded MXNet Java package jar file under the `java-demo` folder.
+You can find nightly release version from [here](https://repository.apache.org/#nexus-search;gav~org.apache.mxnet~~1.3.1-SNAPSHOT~~).
```

Comment:
Is the user needing to do anything with this repo anymore? I don't think they do. Maybe we should move it this information somewhere else. I could see this being potentially confusing since it's the first line in the setup section.